docs/hardware-interfaces/gpio.md

```diff
@@ -2,20 +2,62 @@
 title: GPIO
 ---
 
-# GPIO
+GPIOs (General Purpose Input Outputs) are user-controllable pins that can be used to interact with external circuits.
 
-The GPIOs can be accessed through the GPIO sysfs interface, see the [documentation](https://www.kernel.org/doc/Documentation/gpio/sysfs.txt) for details. 
+Let's get started 🚀
 
-> The GPIO sysfs interface is deprecated but is currently the best option for userspace GPIO access. More context available in [this post by Luz on the Onion Community](https://community.onion.io/topic/4892/can-bus-using-mcp2515-with-omega2/13).
+## Hardware
 
-An important note: because of changes in the kernel, the GPIOs are not numbered like they were before:
+Table listing number of GPIOs on Omega2/2+ vs Omega2S/Omega2S+
 
-- GPIO 0 - 31 ⇒ GPIO 480 - 511  (GPIO n + 480)
-- GPIO 32 - 63 ⇒ GPIO 448 - 479 (GPIO n + 416)
+### Electrical Ratings
 
-## `gpio-lookup` utility
+GPIO Operating Voltage:
 
-To simplify Omega2 GPIO mapping a new utility `gpio-lookup` has been introduced, which would generate equivalent kernel GPIO numbers against given an "actual" GPIO number.
+| Parameter  | Minimum (Vdc) | Minimum (Vdc) |  
+|------------|---------------|---------------|
+|Input HIGH  | 2.0           | 3.6           |
+|Input Low   |-0.3           | 0.8           |
+|Output HIGH | 2.4           | 3.3           |
+|Output LOW  | -             | 0.4           |
+
+:::note 
+The Omega2/2+ GPIOs are not 5V input tolerant!
+
+:::
+
+:::warning
+Connecting a signal to an input pin below the minimum LOW or above the maximum HIGH voltages may damage your Omega device!
+
+:::
+
+Standard 5V logic devices typically accept 3.3V as a logical HIGH, however, the output logical HIGH is in the range of 4.4V to 5V. This means the Omega can output to a 5V logical device, but input from the 5V logic device would damage the GPIO input circuitry.
+
+### Multiplexed Pins
+
+A number of the available pins can be used for multiple purposes other than general-purpose input/output when needed. Such pins are referred to as multiplexed pins.
+
+:::tip
+See detailed explanation of [**Multiplexed Pins**](https://documentation.onioniot.com/hardware-interfaces/pin-multiplexing).
```

Comment:
Please update this link to use file path. for more info see: https://docusaurus.io/docs/2.x/markdown-features/links